app/controllers/api/functions.php

```diff
@@ -1333,7 +1505,7 @@
 
 // Variables
 
-App::post('/v1/functions/:functionId/variables')
+    App::post('/v1/functions/:functionId/variables')
```

Comment:
Why did we change the indentation here and in other similar places? 
```suggestion
App::post('/v1/functions/:functionId/variables')
```

Reply:
I auto-formatted using prettier. I will use `composer format` instead.